b2sdk/encryption/setting.py

```diff
@@ -20,15 +20,18 @@
 
 logger = logging.getLogger(__name__)
 
+UNKNOWN_KEY_ID = ...
```

Comment:
I don't think that `Ellipsis` should be used as sentinel - I think that the current consensus is:
```
import enum 
class Empty(enum.Enum): 
     token = 0

SENTINEL = Empty.token
def something(default: Union[str, Empty] = SENTINEL):
    if default is SENTINEL:
        ...
```
https://github.com/python/typing/issues/236